addons/mail/static/src/components/rtc_option_list/rtc_option_list.scss

```diff
@@ -1,48 +1,19 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_RtcOptionList {
-    display: flex;
-    flex-direction: column;
-}
-
-.o_RtcOptionList_button {
-    display: flex;
-    margin: map-get($spacers, 1);
-    align-items: center;
-    color: gray('800');
-}
-
-.o_RtcOptionList_buttonIcon {
-    margin: map-get($spacers, 1);
-    padding: map-get($spacers, 1);
-}
-
-.o_RtcOptionList_buttonText {
-    @include text-truncate();
-}
-
 // ------------------------------------------------------------------
 // Style
 // ------------------------------------------------------------------
 
 .o_RtcOptionList_button {
     background: none;
-    border: none;
     outline: none;
```

Comment:
To be able to remove the `background` and `outline` properties, and the `@include` below,  you could use `btn` which uses these properties.
Therefore, it will add unwanted padding and font-weight, so you could just add `py-0 px-2 font-weight-normal` to keep the same result on each button 👍 .